### PR TITLE
Add variables to configure a bit php

### PR DESCRIPTION
**Pull request by MiLk**

### CODE REVIEW

File: tasks/centos.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@
     - php{{ php_version | replace('.','') }}-php-pgsql
     - php{{ php_version | replace('.','') }}-php-pdo
     - php{{ php_version | replace('.','') }}-php-mbstring
+    - php{{ php_version | replace('.','') }}-php-xml
```
Comment (christopherobin) [inline tasks/centos.yml:14]: Why the random PHP extension here? Needed by your project?

Comment (MiLk) [inline tasks/centos.yml:14]: I think it was for composer. Also required for another project.
If not required by composer, I will remove it.

Comment (christopherobin) [inline tasks/centos.yml:14]: `composer` uses json for everything so it might be coming from a project, in that case I'd recommend using #3 for XML

**Comment by christopherobin** [comment]
:+1:

**Pull request merged**